CHANGELOG.md

```diff
@@ -2,12 +2,16 @@
 
 This project follows [Semantic Versioning](http://semver.org/).
 
-## [1.0.0] 2021-03-22
+## [2.0.0] 2021-10-01
 ### Changed
-- Variable support for string variables
-- JavaScript function support (lowercase)
+- Changed package name to `easy-path-expressions`.
 
 ## [1.1.0] 2021-04-13
 - Support for logical operators
 - Added operator validation
-- Fixed bugs in variable idenfication
\ No newline at end of file
+- Fixed bugs in variable identification
+
+## [1.0.0] 2021-03-22
+### Changed
+- Variable support for string variables
```

Comment:
Did you add these lines for version 1.0.0?

Reply:
I did not. I reversed the order of the changelog and fixed a typo.